boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -712,11 +742,17 @@ pub unsafe fn main() {
         ),
         i2c_master_slave,
         spi_controller,
+        aes,
         scheduler,
         systick: cortexm4::systick::SysTick::new_with_calibration(64000000),
     };
 
     let _ = platform.pconsole.start();
+
+    // test::aes_test::run_aes128_ctr(&base_peripherals.ecb);
+    // test::aes_test::run_aes128_cbc(&base_peripherals.ecb);
+    // test::aes_test::run_aes128_ecb(&base_peripherals.ecb);
```

Comment:
Why not use the unit test framework to run these instead of commenting them out?

Reply:
That's the right thing to do but not something I'm going to take on right now.